CorsixTH/Src/th_gfx.cpp

```diff
@@ -308,104 +308,104 @@ inline static void _setmin(int& iLeft, int iRight)
 {
     static const uint8_t aHdr[] = {'C', 'T', 'H', 'G', 1, 2};
 
-    if (!input.Available(6))
+    if (!input.are_bytes_available(6))
         return false;
     for (int i = 0; i < 6; i++)
     {
-        if (input.Uint8() != aHdr[i])
+        if (input.read_uint8() != aHdr[i])
             return false;
     }
     return true;
 }
 
-size_t THAnimationManager::loadElements(Input &input, THSpriteSheet *pSpriteSheet,
+size_t animation_manager::load_elements(memory_reader &input, sprite_sheet *pSpriteSheet,
                                         size_t iNumElements, size_t &iLoadedElements,
                                         size_t iElementStart, size_t iElementCount)
 {
     size_t iFirst = iLoadedElements + iElementStart;
 
-    size_t iSpriteCount = pSpriteSheet->getSpriteCount();
+    size_t iSpriteCount = pSpriteSheet->get_sprite_count();
     while (iNumElements > 0)
     {
-        if (iLoadedElements >= iElementCount || !input.Available(12))
+        if (iLoadedElements >= iElementCount || !input.are_bytes_available(12))
             return SIZE_MAX;
 
-        size_t iSprite = input.Uint32();
-        int iX = input.Int16();
-        int iY = input.Int16();
-        uint8_t iLayerClass = input.Uint8();
-        uint8_t iLayerId = input.Uint8();
-        uint32_t iFlags = input.Uint16();
+        size_t iSprite = input.read_uint32();
+        int iX = input.read_int16();
+        int iY = input.read_int16();
+        uint8_t iLayerClass = input.read_uint8();
+        uint8_t iLayerId = input.read_uint8();
+        uint32_t iFlags = input.read_uint16();
 
         if (iLayerClass > 12)
             iLayerClass = 6; // Nothing lives on layer 6
 
-        element_t oElement;
-        oElement.iSprite = iSprite;
-        oElement.iFlags = iFlags;
-        oElement.iX = iX;
-        oElement.iY = iY;
-        oElement.iLayer = iLayerClass;
-        oElement.iLayerId = iLayerId;
-        if (oElement.iSprite >= iSpriteCount)
-            oElement.pSpriteSheet = nullptr;
+        element oElement;
+        oElement.sprite = iSprite;
+        oElement.flags = iFlags;
+        oElement.x = iX;
+        oElement.y = iY;
+        oElement.layer = iLayerClass;
+        oElement.layer_id = iLayerId;
+        if (oElement.sprite >= iSpriteCount)
+            oElement.element_sprite_sheet = nullptr;
         else
-            oElement.pSpriteSheet = pSpriteSheet;
+            oElement.element_sprite_sheet = pSpriteSheet;
 
-        m_vElements.push_back(oElement);
+        elements.push_back(oElement);
         iLoadedElements++;
         iNumElements--;
     }
     return iFirst;
 }
 
-size_t THAnimationManager::makeListElements(size_t iFirstElement, size_t iNumElements,
+size_t animation_manager::make_list_elements(size_t iFirstElement, size_t iNumElements,
                                             size_t &iLoadedListElements,
                                             size_t iListStart, size_t iListCount)
```

Comment:
Indenting of other arguments is incorrect now

Reply:
Ok, it'll get clobbered in the format PR by I will fix it now too.